edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/storage/MessageStore.cs

```diff
@@ -192,25 +192,28 @@ public MessageWrapper(IMessage message, DateTime timeStamp, int refCount)
         class CleanupProcessor : IDisposable
         {
             const int CleanupBatchSize = 10;
-            static readonly TimeSpan CleanupTaskFrequency = TimeSpan.FromMinutes(30); // Run once every 30 mins.
-            static readonly TimeSpan MinCleanupSleepTime = TimeSpan.FromSeconds(30); // Sleep for 30 secs
+            static readonly TimeSpan DefaultCleanupTaskFrequency = TimeSpan.FromMinutes(30); // Check every 30 min that it is still running
```

Comment:
>DefaultCleanupTaskFrequency [](start = 37, length = 27)

nit: this should probably be renamed to avoid future confusion, right now the naming implies that the cleanup task runs every 30min.  Maybe change it to something like CleanupTaskCheckFrequency